### ISSUE TITLE
Collaborative FAQ

### ISSUE THREAD

**Issue by slarrain** [mylar3/mylar3#1098]
The Mylar developers are very active on [Discord](https://discord.gg/6UG94R7E8T) and help people with multiple questions. Some of this questions are asked repeatedly by different people. Making a collaborative FAQ could help solving this doubts without taking time from the developers.

---

**- Mylar can't find a comic on DDL, even though the comic name, when searched on [GetComics.info](GetComics.info) gives only 1 result with a Direct Download link**

Searches done by Mylar on DDL not only include the comic's name, but can also include the year and the comic type.  Sometimes changing the comic type can make it get snatched and downloaded. For example, from One-Shot to TPB, : `Edit Settings` --> `Force-Type (original:One-Shot)` --> `TPB`. Or sometimes checking `Accept all booktype search matches` can make it work.
![image](https://user-images.githubusercontent.com/9893834/158106353-f380d201-a3f7-4b52-b1c9-8f695e27d398.png)

---

**- I selected `Year Removal` on `Edit Settings` on my comic, yet it didn't do anything when searching for downloads.**

`Year Removal` only works for `print` volumes. If it's a non print, usually there's no issue number so it uses the year to help determine if it's provided.

---

**- The _ComicVine_ name of my comic yields no results from DDL. But if I could change the name of the comic it would get a match on DDL**

You can use the `Alternate Search Names` on `Edit Settings` to set up an alternative name to be used when searching for that issue on DDL.

**Comment by evilhero** [comment]
All of the above have now been added to the wiki [FAQ](https://github.com/mylar3/mylar3/wiki/FAQ).

Thanks for getting it all started!

**Comment by slarrain** [comment]
**- What does "Snatched" mean?**

When you add a series, the first stage for an issue is wanted. The searcher goes looking for it from torrent/indexer sites or DDL. Once it finds the comic it sends the torrent to your torrent client / nzb to your downloader / link to the DDL queue. Once that send is complete, the issue is considered snatched.

Depending on which of the three approaches are being used (torrents/nzbs/ddls), the file will end up getting post-processed at some point - after which it will be marked as downloaded.

**Comment by barbequesauce** [comment]
Closing due to staleness, we welcome contributions for new features. If this is an issue and you are still experiencing it, please open a new ticket with a care package attached.
